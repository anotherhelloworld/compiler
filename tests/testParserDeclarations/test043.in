const
a = array[1..10] of 10;