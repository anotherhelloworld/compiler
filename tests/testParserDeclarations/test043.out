Illegal expression: expected 'IDENTIFIER' (2, 5)
